model/service.go

```diff
@@ -169,3 +176,36 @@ func ParseServiceString(s string) *Service {
 		Tags:      tags,
 	}
 }
+
+func (t Tag) String() string {
+	labels := make([]string, 0)
+	for k, v := range t {
+		labels = append(labels, fmt.Sprintf("%s=%s", k, v))
+	}
```

Comment:
would it make sense for the string representation of a key without value to be "key", instead of "key="? Then we could also allow simple tags like "v1", "prod", etc.